lumispy/signals/luminescence_spectrum.py

```diff
@@ -449,26 +458,34 @@ def to_invcm_relative(self, laser=None, inplace=True, jacobian=True):
             self.axes_manager.set_axis(
                 invcmaxis, self.axes_manager.signal_axes[0].index_in_axes_manager
             )
+            self.data = self.isig[::-1].data
             # replace variance axis
             if self.metadata.has_item(
                 "Signal.Noise_properties.variance"
             ) and not isinstance(self.get_noise_variance(), (float, int)):
                 self.metadata.Signal.Noise_properties.variance.axes_manager.set_axis(
                     invcmaxis, self.axes_manager.signal_axes[0].index_in_axes_manager
                 )
+                self.metadata.Signal.Noise_properties.variance.data = (
+                    self.metadata.Signal.Noise_properties.variance.isig[::-1].data
+                )
         # create and return new signal
         else:
             s2 = self.to_invcm(inplace=inplace, jacobian=jacobian)
             s2.axes_manager.set_axis(
                 invcmaxis, self.axes_manager.signal_axes[0].index_in_axes_manager
             )
+            s2.data = s2.isig[::-1].data
```

Comment:
Is there a way to not have to repeat the same lines of code on both sides of the if statement (code not DRY)? What if you set your copy as your signal in this case of inplace=True?

Reply:
Probably an if/else statement could decide whether to operate on a copy of the signal or a pointer to self ... I would however leave that consolidation to a separate PR. I will open an issue.